vscode/package.json

```diff
@@ -466,53 +453,39 @@
           "default": [],
           "description": "Array of filepaths to additional rules for analysis",
           "scope": "window",
-          "order": 5
+          "order": 4
         },
         "konveyor.analysis.labelSelector": {
           "type": "string",
           "default": "discovery",
           "description": "Expression to select incidents based on custom variables. Example: (!package=io.konveyor.demo.config-utils)",
           "scope": "window",
-          "order": 6
-        },
-        "konveyor.analysis.analyzeKnownLibraries": {
-          "type": "boolean",
-          "default": false,
-          "description": "Whether analysis should include known open-source libraries",
-          "scope": "window",
-          "order": 7
-        },
-        "konveyor.analysis.analyzeDependencies": {
-          "type": "boolean",
-          "default": true,
-          "description": "Whether analysis should include dependencies",
-          "scope": "window",
-          "order": 8
+          "order": 5
         },
         "konveyor.analysis.analyzeOnSave": {
           "type": "boolean",
           "default": true,
           "description": "Whether analysis of file should be run when saved",
           "scope": "window",
-          "order": 9
+          "order": 6
         },
         "konveyor.diffEditorType": {
           "type": "string",
+          "enum": [
+            "diff",
+            "merge"
+          ],
           "default": "diff",
           "description": "Diff editor to use when resolving proposed solutions",
-          "scope": "window"
-        },
-        "konveyor.kai.backendURL": {
-          "type": "string",
-          "default": "0.0.0.0:8080",
-          "description": "Kai Backend URL",
-          "scope": "window"
+          "scope": "window",
+          "order": 7
         },
         "konveyor.kai.providerName": {
           "type": "string",
           "default": "ChatIBMGenAI",
           "description": "URL for the Kai solution server",
```

Comment:
This description isn't right yeah? Also, I'm told IBM model doesn't work anymore; can we change the default?

Reply:
Ah this stuff (the kai configurations) will be pulled out to an external file in followup PR later today/tomorrow.

Reply:
But I'll read the descriptions over again and make some adjustments.

Reply:
Updated.